rinex/src/navigation/ephemeris.rs

```diff
@@ -434,12 +444,13 @@ impl Ephemeris {
  */
 fn parse_orbits(
     version: Version,
+    msg: NavMsgType,
     constell: Constellation,
     lines: std::str::Lines<'_>,
 ) -> Result<HashMap<String, OrbitItem>, Error> {
     // Determine closest standards from DB
     // <=> data fields to parse
-    let nav_standards = match closest_nav_standards(constell, version, NavMsgType::LNAV) {
+    let nav_standards = match closest_nav_standards(constell, version, msg) {
```

Comment:
introduce new argument : to also match on NAV frame type.
Truly only used in `v4::parse_orbits()`